### PR TITLE
ByRole: Remove MatchOptions

### PR DESCRIPTION
**Pull request by eps1lon**
Don't merge before https://github.com/testing-library/dom-testing-library/pull/1211 is released in `latest`

Docs for https://github.com/testing-library/dom-testing-library/pull/1211

### CODE REVIEW

File: docs/queries/byrole.mdx [hunk 1]
```diff
@@ -15,13 +15,11 @@ import TabItem from '@theme/TabItem'
...
+  role: string,
   options?: {
-    exact?: boolean = true,
     hidden?: boolean = false,
     name?: TextMatch,
     description?: TextMatch,
-    normalizer?: NormalizerFn,
     selected?: boolean,
     checked?: boolean,
     pressed?: boolean,
```
Comment (timdeschryver) [inline docs/queries/byrole.mdx:25]: While I was looking at the PR I noticed that the docs are missing the `suggest` option.
Could we add it here, or should I create a PR for this?

```suggestion
    pressed?: boolean,
    suggest?: boolean,
```

Comment (eps1lon) [reply docs/queries/byrole.mdx:25]: can you do this in a separate one? I don't actually know what this option does :D

**Review by timdeschryver — APPROVED** [review]
Approving already so this can be merged whenever the release will be

File: docs/queries/byrole.mdx (hunk 1 above)
Comment (MatanBobi) [reply docs/queries/byrole.mdx:25]: @timdeschryver Do you want me to take it? I have some free time :)

Comment (timdeschryver) [reply docs/queries/byrole.mdx:25]: Sure, go for it @MatanBobi

**Pull request merged**